src/res/translation/translation_nl_NL.ts

```diff
@@ -414,7 +414,7 @@
     <message>
         <location filename="../../audiomixerboard.cpp" line="233"/>
         <source>PAN</source>
-        <translation>BAL</translation>
+        <translation>BAL.</translation>
```

Comment:
Same as above.

Reply:
Correct here as well, I had second thoughts about this and the . (while it does not look nice in the GUI) should be there as it is an abbrevation and this is in-line with the text when showed in normal and compact skin.